lib/packwerk/cli.rb

```diff
@@ -57,7 +57,11 @@ def execute_command(args)
       when "detect-stale-violations"
         output_result(parse_run(args).detect_stale_violations)
       when "update-deprecations"
-        output_result(parse_run(args).update_deprecations)
+        output_result(parse_run(args).update_todo)
+      when "update-todo"
+        output_result(parse_run(args).update_todo)
+      when "todo"
```

Comment:
Todo isn't a good shorthand because it isn't a verb. It should be `update`.

Reply:
Alrighty I changed it back. I felt like Matt C. had some convincing points here that I think are worth responding to, but would love to be able to move forward on this too.